ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -522,8 +522,9 @@ protected Boolean findNextCompactionAndExecute(boolean collectGenericStats, bool
           UserGroupInformation ugi = UserGroupInformation.createProxyUser(ci.runAs,
               UserGroupInformation.getLoginUser());
 
+          ValidCompactorWriteIdList finalTblValidWriteIds = tblValidWriteIds;
```

Comment:
did something change here?

Reply:
I reverted it, it was remainder of the former solution approach